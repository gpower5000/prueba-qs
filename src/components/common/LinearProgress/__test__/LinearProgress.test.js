
import React from 'react';
import { LinearProgress } from '../index';
import renderer from 'react-test-renderer';

describe('Component: LinearProgress', () => {
    it('<> Render Correctly </>', () => {
        const tree = renderer.create(<LinearProgress />).toJSON();
        expect(tree).toMatchSnapshot();
    })
    it('<> Render size small </>', () => {
        const tree = renderer.create(<LinearProgress size = 'small' />).toJSON();
        expect(tree).toMatchSnapshot();
    })
    it('<> Render size normal </>', () => {
        const tree = renderer.create(<LinearProgress size = 'normal' />).toJSON();
        expect(tree).toMatchSnapshot();
    })
    it('<> Render size large </>', () => {
        const tree = renderer.create(<LinearProgress size = 'large' />).toJSON();
        expect(tree).toMatchSnapshot();
    })
})
