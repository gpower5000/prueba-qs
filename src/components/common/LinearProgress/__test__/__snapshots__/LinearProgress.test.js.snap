// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: LinearProgress <> Render Correctly </> 1`] = `
<div
  className="c-linear-progress_content c-linear-progresss__size-small c-linear-progresss__color-primary"
>
  <div
    className="c-linear-progress_indeterminate-one"
  />
  <div
    className="c-linear-progress_indeterminate-two"
  />
</div>
`;

exports[`Component: LinearProgress <> Render size large </> 1`] = `
<div
  className="c-linear-progress_content c-linear-progresss__size-large c-linear-progresss__color-primary"
>
  <div
    className="c-linear-progress_indeterminate-one"
  />
  <div
    className="c-linear-progress_indeterminate-two"
  />
</div>
`;

exports[`Component: LinearProgress <> Render size normal </> 1`] = `
<div
  className="c-linear-progress_content c-linear-progresss__size-normal c-linear-progresss__color-primary"
>
  <div
    className="c-linear-progress_indeterminate-one"
  />
  <div
    className="c-linear-progress_indeterminate-two"
  />
</div>
`;

exports[`Component: LinearProgress <> Render size small </> 1`] = `
<div
  className="c-linear-progress_content c-linear-progresss__size-small c-linear-progresss__color-primary"
>
  <div
    className="c-linear-progress_indeterminate-one"
  />
  <div
    className="c-linear-progress_indeterminate-two"
  />
</div>
`;
